app/code/core/Mage/Adminhtml/Block/Catalog/Category/Tree.php

```diff
@@ -242,7 +242,7 @@ protected function _getNodeJson($node, $level = 0)
         $allowMove = $this->_isCategoryMoveable($node);
         $item['allowDrop'] = $allowMove;
         // disallow drag if it's first level and category is root of a store
-        $item['allowDrag'] = $allowMove && (($node->getLevel()==1 && $rootForStores) ? false : true);
+        $item['allowDrag'] = $allowMove && (!($node->getLevel() == 1 && $rootForStores));
```

Comment:
Unnecessary brackets?

```suggestion
$item['allowDrag'] = $allowMove && !($node->getLevel() == 1 && $rootForStores);
```